osbenchmark/worker_coordinator/runner.py

```diff
@@ -652,6 +653,159 @@ def __repr__(self, *args, **kwargs):
         return "bulk-index"
 
 
+class DeleteKnnModel(Runner):
+    """
+    Deletes the K-NN model named model_id.
+    """
+
+    NAME = "delete-knn-model"
+
+    async def __call__(self, opensearch, params):
+        model_id = parse_string_parameter("model_id", params)
```

Comment:
did you check can we use mandatory?

Reply:
Yes, these helpers enforce mandatory if they don't have a third argument. So if model_id is not provided `parse_string_parameter` will throw an error.